lib/messages/create/create-issue-dialog.js

```diff
@@ -29,5 +29,19 @@ module.exports = (repository) => {
       },
     ],
   };
+
+  if (Array.isArray(labels)) {
```

Comment:
this is going to be true even if `labels = []` right?

Should this be something like `if (labels.length !== 0)` ?